<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База данных конструкционных материалов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="leFilter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbApplyFilter">
            <property name="text">
             <string>Применить фильтр</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>328</width>
            <height>0</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>362</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="mplLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="actionNewDB"/>
    <addaction name="actionOpenDB"/>
    <addaction name="actionSaveDB"/>
    <addaction name="actionSaveDBas"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Материал</string>
    </property>
    <addaction name="actionAddMaterial"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Графическое окно</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionVarSetup"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDB"/>
   <addaction name="actionSaveDB"/>
   <addaction name="actionSaveDBas"/>
   <addaction name="actionOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actionAddMaterial"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>71</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAddMaterial">
   <property name="text">
    <string>Добавить материал</string>
   </property>
   <property name="toolTip">
    <string>Добавить материал в базу</string>
   </property>
  </action>
  <action name="actionSaveDB">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить базу данных</string>
   </property>
  </action>
  <action name="actionOpenDB">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="actionNewDB">
   <property name="text">
    <string>Очистить БД</string>
   </property>
   <property name="toolTip">
    <string>Удалить все записи из базы данных</string>
   </property>
  </action>
  <action name="actionSaveDBas">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="toolTip">
    <string>Сохранить базу данных как</string>
   </property>
  </action>
  <action name="actionMPLsetup">
   <property name="text">
    <string>Настройка вида</string>
   </property>
   <property name="toolTip">
    <string>Настройка вида графиков</string>
   </property>
  </action>
  <action name="actionSavePNG">
   <property name="text">
    <string>Сохранить картинку</string>
   </property>
   <property name="toolTip">
    <string>Сохранить графики в виде картинки</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Настройка отображения переменных</string>
   </property>
  </action>
  <action name="actionVarSetup">
   <property name="text">
    <string>Настройка отображения переменных</string>
   </property>
   <property name="toolTip">
    <string>Настройка отображения переменных</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ic.qrc"/>
 </resources>
 <connections/>
</ui>
